testapps/ClassLibraryWithPrecompiledViews/ClassLibraryWithPrecompiledViews.csproj

```diff
@@ -1,15 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
```

Comment:
We don't need tests to verify if precompilation tool works for "class libraries". It did not work very well in the past. 

That said, the test failure here was that our most recent changes discovers compiled views from the .Views dll before .PrecompiledViews.dll which is what was happening here. We could fix the test but it's not very valuable.